tests/unit/metadata/test_metadata.py

```diff
@@ -147,9 +150,10 @@ def test_load_from_json_single_table(self, mock_read_json):
             assert instance.tables[table_name].sequence_index is None
             assert instance.tables[table_name]._version == 'SINGLE_TABLE_V1'
 
+    @patch('builtins.open', new_callable=mock_open)
     @patch('sdv.metadata.utils.Path')
     @patch('sdv.metadata.utils.json')
-    def test_load_from_json_multi_table(self, mock_json, mock_path):
+    def test_load_from_json_multi_table(self, mock_json, mock_path, mock_file):
```

Comment:
This test was failing when run standalone -- With this we fix the issue of running just this file as test.